From a105ddc4136df2b0d8f6b143a5e46d353a5ea128 Mon Sep 17 00:00:00 2001
From: Ken Dreyer <ktdreyer@ktdreyer.com>
Date: Mon, 8 Oct 2012 21:01:52 -0600
Subject: [PATCH 5/5] systemd: run "sysnames" script after starting afsd

This helper script will set any custom sysnames on the client.

Change-Id: Ibdb770fac20eb26b137bd02a71a52e0c801b039d
---
 src/packaging/RedHat/openafs-client.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/packaging/RedHat/openafs-client.service b/src/packaging/RedHat/openafs-client.service
index 719aed77..aac1ca7 100644
--- a/src/packaging/RedHat/openafs-client.service
+++ b/src/packaging/RedHat/openafs-client.service
@@ -7,6 +7,7 @@ Type=forking
 EnvironmentFile=/etc/sysconfig/openafs
 ExecStartPre=/sbin/modprobe openafs
 ExecStart=/usr/sbin/afsd $AFSD_ARGS $CACHE $OPTIONS $CONFDIR
+ExecStartPost=/usr/libexec/openafs/sysnames
 ExecStop=/bin/umount /afs
 ExecStop=/usr/sbin/afsd -shutdown
 ExecStop=/sbin/rmmod libafs
-- 
1.7.11.7

